sormas-ui/src/main/java/de/symeda/sormas/ui/events/eventParticipantLink/EventParticipantListEntry.java

```diff
@@ -80,7 +82,31 @@ public EventParticipantListEntry(EventParticipantListEntryDto eventParticipantLi
 		titleLabel.setDescription(eventParticipantListEntryDto.getEventTitle());
 		mainLayout.addComponent(titleLabel);
 
-		mainLayout.addComponent(titleLabel);
+		if (eventParticipantListEntryDto.getEventStartDate() != null) {
+			HorizontalLayout eventDateLayout = new HorizontalLayout();
```

Comment:
There is already a format for date of event on Event info cards present on CaseDataView and ContactDataView. Please use the same format.